src/test/java/uk/gov/companieshouse/certifiedcopies/orders/api/mapper/CertifiedCopyItemMapperTest.java

```diff
@@ -0,0 +1,186 @@
+                is(dto.getItemOptions().getFilingHistoryDocuments().get(0).getFilingHistoryId()));
+    }
+
+    @Test
+    public void testCertifiedCopyItemRequestDTOToCertifiedCopyItemDefaults() {
+        final CertifiedCopyItemRequestDTO dto = new CertifiedCopyItemRequestDTO();
+        dto.setItemOptions(new CertifiedCopyItemOptionsRequestDTO());
+
+        final CertifiedCopyItem certifiedCopyItem = mapperUnderTest.certifiedCopyItemRequestDTOToCertifiedCopyItem(dto);
+
+        CertifiedCopyItemData item = certifiedCopyItem.getData();
+        assertThat(item.getItemOptions().getDeliveryMethod(), is(DeliveryMethod.POSTAL));
+        assertThat(item.getItemOptions().getDeliveryTimescale(), is(DeliveryTimescale.STANDARD));
+    }
+
+    @Test
+    public void testCertifiedCopyItemDataToCertifiedCopyItemResponseDTO() {
+        final CertifiedCopyItemData item = new CertifiedCopyItemData();
+        item.setId(ID);
+        item.setCompanyName(COMPANY_NAME);
+        item.setCompanyNumber(COMPANY_NUMBER);
+        item.setCustomerReference(CUSTOMER_REFERENCE);
+        item.setDescription(DESCRIPTION);
+        item.setDescriptionIdentifier(DESCRIPTION_IDENTIFIER);
+        item.setDescriptionValues(DESCRIPTION_VALUES);
+        item.setEtag(ETAG);
+        item.setItemCosts(ITEM_COSTS);
+        item.setItemOptions(ITEM_OPTIONS);
+        item.setKind(KIND);
+        item.setLinks(LINKS);
+        item.setPostageCost(POSTAGE_COST);
+        item.setPostalDelivery(IS_POSTAL_DELIVERY);
+        item.setQuantity(QUANTITY);
+        item.setTotalItemCost(TOTAL_ITEM_COST);
+
+        final CertifiedCopyItemResponseDTO dto = mapperUnderTest.certifiedCopyItemDataToCertifiedCopyItemResponseDTO(item);
+
+        assertThat(dto.getId(), is(item.getId()));
+        assertThat(dto.getCompanyName(), is(item.getCompanyName()));
+        assertThat(dto.getCompanyNumber(), is(item.getCompanyNumber()));
+        assertThat(dto.getCustomerReference(), is(item.getCustomerReference()));
+        assertThat(dto.getDescription(), is(item.getDescription()));
+        assertThat(dto.getDescriptionIdentifier(), is(item.getDescriptionIdentifier()));
+        assertThat(dto.getDescriptionValues(), is(item.getDescriptionValues()));
+        assertThat(dto.getEtag(), is(item.getEtag()));
+        assertThat(dto.getItemCosts(), is(item.getItemCosts()));
+
+        assertThat(dto.getItemOptions().getCollectionLocation(), is(item.getItemOptions().getCollectionLocation()));
+        assertThat(dto.getItemOptions().getContactNumber(), is(item.getItemOptions().getContactNumber()));
+        assertThat(dto.getItemOptions().getDeliveryMethod(), is(item.getItemOptions().getDeliveryMethod()));
+        assertThat(dto.getItemOptions().getDeliveryTimescale(), is(item.getItemOptions().getDeliveryTimescale()));
+        assertThat(dto.getItemOptions().getFilingHistoryDocuments().get(0), is(item.getItemOptions().getFilingHistoryDocuments().get(0)));
```

Comment:
Could this class be formatted to match CH Java coding standards please? https://github.com/companieshouse/styleguides/blob/master/java.md